ROR-Assignment-Day-04/economic_analysis.rb

```diff
@@ -0,0 +1,46 @@
+# Define module DebtGdpRatio
+module DebtGdpRatio
+    # Method to calculate debt-to-GDP ratio
+  def self.debt_gdp_ratio(gdp, debt)
+    return 'GDP cannot be zero' if gdp == 0
+    (debt.to_f / gdp.to_f) * 100
+  end
+end
+  
+  # Define module EconomicAnalysis
+module EconomicAnalysis
```

Comment:
After moving `DebtGdpRatio` into separate file you can include that in this file like below. 
```suggestion
module EconomicAnalysis
    include DebtGdpRatio
```